pennylane/_qubit_device.py

```diff
@@ -1139,6 +1154,71 @@ def mutual_info(self, wires0, wires1, log_base):
             state, indices0=wires0, indices1=wires1, c_dtype=self.C_DTYPE, base=log_base
         )
 
+    def classical_shadow(self, wires, n_snapshots, circuit, seed=None):
```

Comment:
where in the docs is this displayed? I only found the base module `classical_shadow` function and the method in `default.qubit`

Reply:
The documentation for `QubitDevice` is here: https://pennylane--2820.org.readthedocs.build/en/2820/code/api/pennylane.QubitDevice.html#pennylane.QubitDevice.classical_shadow